library/src/main/java/io/voucherify/android/client/module/VoucherModule.java

```diff
@@ -17,25 +18,16 @@
      * @param trackingId custom tracking id to track voucher consumers
      */
     public VoucherModule(VoucherifyApi api, Scheduler scheduler, String trackingId) {
-        super(api, scheduler, trackingId);
+        this.validation = new Validation(api, scheduler, trackingId);
```

Comment:
:+1: best will be to inject it later when we introduce dagger :) injecting via constructor for now